src/Compilers/CSharp/Test/Syntax/Syntax/FieldAndValueKeywordTests.cs

```diff
@@ -0,0 +1,1241 @@
+                        get { return field; }
+                        set { _ = field; }
+                    }
+                }
+                class B
+                {
+                    object this[int @field]
+                    {
+                        get { return @field; }
+                        set { _ = @field; }
+                    }
+                }
+                """;
+            var comp = CreateCompilation(source, parseOptions: TestOptions.Regular12);
+            // No diagnostics expected for field in indexers.
+            comp.VerifyEmitDiagnostics();
+        }
+
+        [Fact]
+        public void Parameter_02()
+        {
+            string source = """
+                class A
+                {
+                    object this[int value]
+                    {
+                        get { return value; }
+                        set { _ = value; }
+                    }
+                }
+                class B
+                {
+                    object this[int @value]
+                    {
+                        get { return @value; }
+                        set { _ = @value; }
+                    }
+                }
+                """;
+            var comp = CreateCompilation(source, parseOptions: TestOptions.Regular12);
+            comp.VerifyEmitDiagnostics(
+                // (3,21): error CS0316: The parameter name 'value' conflicts with an automatically-generated parameter name
+                //     object this[int value]
+                Diagnostic(ErrorCode.ERR_DuplicateGeneratedName, "value").WithArguments("value").WithLocation(3, 21),
+                // (6,19): error CS0229: Ambiguity between 'int value' and 'object value'
+                //         set { _ = value; }
+                Diagnostic(ErrorCode.ERR_AmbigMember, "value").WithArguments("int value", "object value").WithLocation(6, 19),
+                // (6,19): info CS9248: 'value' is a contextual keyword, with a specific meaning, starting in language version preview. Use '@value' to avoid a breaking change when compiling with language version preview or later.
+                //         set { _ = value; }
+                Diagnostic(ErrorCode.INF_IdentifierConflictWithContextualKeyword, "value").WithArguments("value", "preview").WithLocation(6, 19),
+                // (11,21): error CS0316: The parameter name 'value' conflicts with an automatically-generated parameter name
+                //     object this[int @value]
+                Diagnostic(ErrorCode.ERR_DuplicateGeneratedName, "@value").WithArguments("value").WithLocation(11, 21),
+                // (14,19): error CS0229: Ambiguity between 'int value' and 'object value'
+                //         set { _ = @value; }
+                Diagnostic(ErrorCode.ERR_AmbigMember, "@value").WithArguments("int value", "object value").WithLocation(14, 19));
+        }
+
+        [Theory]
+        [CombinatorialData]
+        public void Event_01(
```

Comment:
>Event_01 [](http://example.com/codeflow?start=20&length=8)

Should `field` and `value` be considered keywords in `event` accessors?

Reply:
I think `value` should be a keyword as that parameter exists in the `add` and `remove` accessors today (see `SynthesizedEventAccessorValueParameterSymbol`), like it does in a property setter (`SynthesizedPropertyAccessorValueParameterSymbol`).
But I don't think `field` will take any particular meaning in events with the C#Next feature, so I don't think it should be a keyword.

What do you think?

Reply:
this matches my takeaway from last LDM on this topic.